src/pkcs15init/pkcs15-westcos.c

```diff
@@ -250,7 +250,7 @@ static int westcos_pkcs15init_generate_key(sc_profile_t *profile,
 	    EVP_PKEY_CTX_set1_rsa_keygen_pubexp(pctx, bn) != 1 ||
 #else
 	    EVP_PKEY_CTX_set_rsa_keygen_pubexp(pctx, bn) != 1 ||
-	    (bn = NULL) || /* pctx will free bn */
+	    (bn = NULL) != NULL || /* pctx will free bn */
```

Comment:
not sure if this will not result in some report "condition is always true". Can we break this long condition to more blocks and use the `bn = NULL;` as normal assignement?